jujugui/static/gui/src/app/components/added-services-list/added-services-list.js

```diff
@@ -24,7 +24,7 @@ YUI.add('added-services-list', function() {
 
     generateItemList: function(services) {
       var items = [];
-      services.forEach(function(service) {
+      services.forEach((service) => {
```

Comment:
no f'ing way...did he actually arrow func? Is that even allowed? That works in IE10?